doc/guide/dynamics/dynamics-stochastic.rst

```diff
@@ -178,21 +183,52 @@ in ``result.measurements``.
     ax.set_xlabel('Time')
     ax.legend()
 
-..
-    TODO merge qutip-tutorials#61
-    For other examples on :func:`qutip.solver.stochastic.smesolve`, see the
-    `following notebook <...>`_, as well as these notebooks available at
-    `QuTiP Tutorials page <https://qutip.org/tutorials.html>`_:
-    `heterodyne detection <...>`_,
-    `inefficient detection <...>`_, and
-    `feedback control <https://github.com/jrjohansson/reproduced-papers/blob/master/Reproduce-SIAM-JCO-46-445-2007-Mirrahimi.ipynb>`_.
+
+Run from known measurements
+===========================
+
+In situations where instead of running multiple trajectories, we want to reproduce a single trajectory from known noise or measurements obtained in lab.
+In these cases, we can use :meth:`~qutip.solver.stochastic.SMESolver.run_from_experiment`.
+
+Let use the measurement output ``J_x`` of the first trajectory of the previous simulation as the input to recompute a trajectory:
+
+.. code-block::
+
+    # Create a stochastic solver instance with the some Hamiltonian as the
+    # previous evolution.
+    solver = SMESolver(
+        H, sc_ops=[np.sqrt(KAPPA) * a],
+        options={"dt": 0.00125, "store_measurement": True,}
+    )
+
+    # Run the evolution, noise
+    recreated_solution = solver.run_from_experiment(
+        rho_0, tlist, stoc_solution.measurements[0],
+        e_ops=[H],
+        # The third parameter is a measurement, not Gaussian noise
+        measurement=True,
+    )
+
+This will recompute the states, expectation values and wiener increments for that trajectory.
+
+.. note::
+
+  The measurements in the result is computed from the state at the end of the time step.
+  However, when using ``run_from_experiment`` with measurement input, the state at the start of the time step is used.
+
+
+For other examples on :func:`qutip.solver.stochastic.smesolve`, see the
+notebooks available at `QuTiP Tutorials page <https://qutip.org/tutorials.html>`_:
+
+`heterodyne detection <https://nbviewer.org/urls/qutip.org/qutip-tutorials/tutorials-v5/time-evolution/015_smesolve-heterodyne.ipynb>`_,
+`inefficient detection <https://nbviewer.org/urls/qutip.org/qutip-tutorials/tutorials-v5/time-evolution/016_smesolve-inefficient-detection.ipynb>`_, and
+`feedback control <https://github.com/jrjohansson/reproduced-papers/blob/master/Reproduce-SIAM-JCO-46-445-2007-Mirrahimi.ipynb>`_.
```

Comment:
```suggestion
notebooks available on the `QuTiP Tutorials page <https://qutip.org/tutorials.html>`_:

* `Heterodyne detection <https://nbviewer.org/urls/qutip.org/qutip-tutorials/tutorials-v5/time-evolution/015_smesolve-heterodyne.ipynb>`_
* `Inefficient detection <https://nbviewer.org/urls/qutip.org/qutip-tutorials/tutorials-v5/time-evolution/016_smesolve-inefficient-detection.ipynb>`_
* `Feedback control <https://github.com/jrjohansson/reproduced-papers/blob/master/Reproduce-SIAM-JCO-46-445-2007-Mirrahimi.ipynb>`_
```

Fixed the formatting a bit. Why are we linking to a notebook in an external repo?

Reply:
We had it in our tutorial even if it was not in our repo... It's no longer there so I will remove it.